src/commonComponents/button/Button.js

```diff
@@ -13,14 +13,21 @@ const TYPES = {
 };
 
 function Button(props) {
-    const { type = TYPES.PRIMARY, className, label } = props;
+    const { type = TYPES.PRIMARY, className, label, handleClick } = props;
```

Comment:
destructure the rest props too and have it with the button to make it more reusable and rename handleClick to onClick